src/style.css

```diff
@@ -1 +1,60 @@
-/* add styles here */
+body {
+  margin: 0;
```

Comment:
This is a minor issue, but according to our checklist, we should avoid using tag names for styling, even for `body`. It's better to add a class to the `body` tag and style it.